### PR TITLE
Add "Using Bundles" section to go/bundle/README

### PR DESCRIPTION
**Pull request by irori**
with caveats for unsigned bundles.

Preview: https://github.com/irori/webpackage/tree/readme/go/bundle#using-bundles

### CODE REVIEW

File: go/bundle/README.md [hunk 1]
```diff
@@ -93,3 +93,52 @@ sign-bundle \
...
 ```
+
+## Using Bundles
+Bundles generated with `gen-bundle` can be opened with web browsers supporting web bundles.
+
+- Chrome (80+): enable `chrome://flags/#web-bundles` flag to enable experimental support for Web Bundles.
+
+## Dealing with Common Problems in Unsigned Bundles
+
+In Chrome's current experimental implementation, bundles can be loaded from local files. Currently only unsigned bundles are supported and they are loaded in "untrusted mode", where:
```
Comment (kinu) [inline go/bundle/README.md]: 'current', 'currently' --> as of Chrome 80 ?

As of Chrome 80, only unsigned bundles are supported and they can only be loaded from local files. Such bundles are loaded as "untrusted" pages, where:

File: go/bundle/README.md [hunk 2]
```diff
@@ -93,3 +93,52 @@ sign-bundle \
 ```
 dump-bundle -i foo.wbn
 ```
+
+## Using Bundles
+Bundles generated with `gen-bundle` can be opened with web browsers supporting web bundles.
+
+- Chrome (80+): enable `chrome://flags/#web-bundles` flag to enable experimental support for Web Bundles.
```
Comment (horo-t) [inline go/bundle/README.md]: All these problems below are only for Chrome.
I think we should have a doc about these information in chromium repository, and this document should have a link to that.
So can we can update the information when we change the implementation synchronously.

How about this?

Chrome (80+) experimentally supports Web Bundles with some limitations. See [this document](https://chromium.googlesource.com/chromium/src/+/refs/heads/master/content/browser/web_package/web_bundles.md) for more details.

Comment (kinu) [reply go/bundle/README.md]: Having this doc in chromium repository and linking from here sounds good, so +1

Comment (irori) [reply go/bundle/README.md]: OK, created https://chromium-review.googlesource.com/c/chromium/src/+/1880413.

**Comment by irori** [comment]
Marked as non substantive for IPR from ash-nazg.

File: go/bundle/README.md (hunk 2 above)
Comment (irori) [reply go/bundle/README.md]: Done

**Pull request merged**